tests/run/phantom-var-2.scala

```diff
@@ -0,0 +1,35 @@
+
+object Test {
+  import Boo._
+
+  def main(args: Array[String]): Unit = {
+    val f = new Foo
+    f.foo
+    f.foo
+    f.foo = {
+      println("foo3")
+      any
+    }
+    f.foo
+    assert(!f.getClass.getDeclaredFields.exists(_.getName.startsWith("foo")), "field foo not erased")
+  }
+}
+
+class Foo {
+  import Boo._
+
+  var foo = {
```

Comment:
I don't think it makes sense to allow phantom `var`. `var` implies runtime semantics that just make no sense for phantoms.